integrations/azure/azure_integration/utils.py

```diff
@@ -0,0 +1,64 @@
+import typing
+
+from port_ocean.context.event import event
+from port_ocean.ocean import ocean
+
+from azure_integration.overrides import AzurePortAppConfig
+
+BATCH_SIZE = 20
+
+
+def get_integration_subscription_id() -> str:
+    logic_settings = ocean.integration_config
+    subscription_id = logic_settings.get("subscription_id", "")
+    return subscription_id
+
+
+async def get_port_resource_configuration_by_kind(kind: str) -> dict:
```

Comment:
Do we still need it?
You can access it using the event

If it is still needed make sure to support a list of kinds

Reply:
it is, for when we receive an event from azure